network/src/dms.rs

```diff
@@ -67,6 +83,133 @@ trait DistributedMessageSetRpcInterface: Send + Sync + 'static {
     ) -> Result<Vec<RawMessage>, String>;
 }
 
+#[async_trait]
+impl<S: Storage> DistributedMessageSetRpcInterface for StorageWrapper<S> {
+    async fn get_message(
+        &self,
+        height: BlockHeight,
+        knowns: Vec<Hash256>,
+    ) -> Result<Vec<RawMessage>, String> {
```

Comment:
Why do you use String as the error instead of anyhow?

Reply:
RPC interface is for network communication. Anyhow::error can't be serialized into a network packet

Reply:
OK